modules/nuxeo-apidoc-core/src/test/resources/markdown/nuxeo-apidoc-core-readme.md

```diff
@@ -1,5 +1,9 @@
 ## About nuxeo-apidoc-core
 
```

Comment:
I should probably add other files use cases on this one and remove the large stream and core-bulk examples in unit tests...

Reply:
Done